r124/inref/ydb312gtm8182b.m

```diff
@@ -0,0 +1,45 @@
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;								;
+; Copyright (c) 2018 YottaDB LLC. and/or its subsidiaries.	;
+; All rights reserved.						;
+;								;
+;	This source code contains the intellectual property	;
+;	of its copyright holder(s), and is made available	;
+;	under a license.  If you do not know the terms of	;
+;	the license, please stop and do not read further.	;
+;								;
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;
+
+ydb231gtm8182b
+	DO getPaths
+
+	set $ZGBLDIR=INST1gbldir
+	set x=$get(^a)
+	set $ZGBLDIR=INST3gbldir
+
+	set incrtrapNODISP=1
+	set $ZTRAP="goto incrtrap^incrtrap"
+	write "$zrealstor BEFORE $ZPEEK loop is ",$zrealstor,!
```

Comment:
$ZPEEK -> $ZPEEK of jnlpool_ctl_struct.ftok_counter_halted

Apply this to lines 23 and 25.